addons/website_sale/controllers/reorder.py

```diff
@@ -12,7 +12,7 @@ def _sale_reorder_get_line_context(self):
         return {}
 
     @route(
-        '/my/orders/reorder_modal_content',
+        '/my/orders/reorder_content',
         type='jsonrpc',
         auth='public',
         website=True,
```

Comment:
I think It would be better to drop this route and handle the reorder process directly in the controllers